Automatic Validation ended with: 
> 2026-04-24T16:29:56.6813822Z ##[error] Manifest: Manifest type not supported. ManifestType: singleton. File: manifests/m/MoltenPixelStudio/LinkDrive/0.2.0/MoltenPixelStudio.LinkDrive.yaml
2026-04-24T16:29:56.6815501Z ##[error] Manifest: Manifest type not supported. ManifestType: singleton. File: manifests/m/MoltenPixelStudio/LinkDrive/0.2.0/MoltenPixelStudio.LinkDrive.yaml

(Automated response - build 1260.)

--- a/manifests/m/MoltenPixelStudio/LinkDrive/0.2.0/MoltenPixelStudio.LinkDrive.installer.yaml
+++ b/manifests/m/MoltenPixelStudio/LinkDrive/0.2.0/MoltenPixelStudio.LinkDrive.installer.yaml
@@ -1,0 +1,22 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+PackageIdentifier: MoltenPixelStudio.LinkDrive
+PackageVersion: 0.2.0
+InstallerType: exe
+Scope: user
+InstallModes:
+  - silent
+  - silentWithProgress
+  - interactive
+InstallerSwitches:
+  Silent: --silent-install
+  SilentWithProgress: --silent-install
+UpgradeBehavior: install
+ReleaseDate: 2026-04-24
+Installers:
+  - Architecture: x64
+    InstallerUrl: https://github.com/MoltenPixelStudio/linkdrive/releases/download/v0.2.0/LinkDrive-Setup-0.2.0-win-x64.exe
+    InstallerSha256: ACF21D97E6329F88B708FA9BD54123B637A73A819D0DA73150E7A423E4213D26
+    InstallerSuccessCodes:
+      - 0
+ManifestType: installer
+ManifestVersion: 1.6.0

--- a/manifests/m/MoltenPixelStudio/LinkDrive/0.2.0/MoltenPixelStudio.LinkDrive.installer.yaml
+++ b/manifests/m/MoltenPixelStudio/LinkDrive/0.2.0/MoltenPixelStudio.LinkDrive.installer.yaml
@@ -1,4 +1,4 @@
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.12.0.schema.json
 PackageIdentifier: MoltenPixelStudio.LinkDrive
 PackageVersion: 0.2.0
 InstallerType: exe
@@ -19,4 +19,4 @@ Installers:
     InstallerSuccessCodes:
       - 0
 ManifestType: installer
-ManifestVersion: 1.6.0
+ManifestVersion: 1.12.0

--- a/manifests/m/MoltenPixelStudio/LinkDrive/0.2.0/MoltenPixelStudio.LinkDrive.locale.en-US.yaml
+++ b/manifests/m/MoltenPixelStudio/LinkDrive/0.2.0/MoltenPixelStudio.LinkDrive.locale.en-US.yaml
@@ -1,4 +1,4 @@
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.12.0.schema.json
 PackageIdentifier: MoltenPixelStudio.LinkDrive
 PackageVersion: 0.2.0
 PackageLocale: en-US
@@ -22,4 +22,4 @@ Tags:
 ReleaseNotes: See https://github.com/MoltenPixelStudio/linkdrive/releases/tag/v0.2.0
 ReleaseNotesUrl: https://github.com/MoltenPixelStudio/linkdrive/releases/tag/v0.2.0
 ManifestType: defaultLocale
-ManifestVersion: 1.6.0
+ManifestVersion: 1.12.0

--- a/manifests/m/MoltenPixelStudio/LinkDrive/0.2.0/MoltenPixelStudio.LinkDrive.locale.en-US.yaml
+++ b/manifests/m/MoltenPixelStudio/LinkDrive/0.2.0/MoltenPixelStudio.LinkDrive.locale.en-US.yaml
@@ -1,0 +1,25 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+PackageIdentifier: MoltenPixelStudio.LinkDrive
+PackageVersion: 0.2.0
+PackageLocale: en-US
+Publisher: MoltenPixelStudio
+PublisherUrl: https://github.com/MoltenPixelStudio
+PublisherSupportUrl: https://github.com/MoltenPixelStudio/linkdrive/issues
+PackageName: LinkDrive
+PackageUrl: https://github.com/MoltenPixelStudio/linkdrive
+License: Proprietary
+LicenseUrl: https://github.com/MoltenPixelStudio/linkdrive/blob/main/LICENSE
+ShortDescription: Fast cross-device file explorer with VPS + LAN peer access
+Description: |-
+  LinkDrive is a fast file explorer for Windows with first-class SFTP / VPS browsing,
+  multi-select transfers with cancel/resume, folder mirroring, bookmarks, SSH config
+  import, and Tailscale peer discovery.
+Tags:
+  - file-manager
+  - sftp
+  - ssh
+  - file-transfer
+ReleaseNotes: See https://github.com/MoltenPixelStudio/linkdrive/releases/tag/v0.2.0
+ReleaseNotesUrl: https://github.com/MoltenPixelStudio/linkdrive/releases/tag/v0.2.0
+ManifestType: defaultLocale
+ManifestVersion: 1.6.0

--- a/manifests/m/MoltenPixelStudio/LinkDrive/0.2.0/MoltenPixelStudio.LinkDrive.yaml
+++ b/manifests/m/MoltenPixelStudio/LinkDrive/0.2.0/MoltenPixelStudio.LinkDrive.yaml
@@ -1,6 +1,6 @@
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.12.0.schema.json
 PackageIdentifier: MoltenPixelStudio.LinkDrive
 PackageVersion: 0.2.0
 DefaultLocale: en-US
 ManifestType: version
-ManifestVersion: 1.6.0
+ManifestVersion: 1.12.0

--- a/manifests/m/MoltenPixelStudio/LinkDrive/0.2.0/MoltenPixelStudio.LinkDrive.yaml
+++ b/manifests/m/MoltenPixelStudio/LinkDrive/0.2.0/MoltenPixelStudio.LinkDrive.yaml
@@ -1,50 +1,6 @@
-# Winget package manifest (v1 singleton). Submit via
-# https://github.com/microsoft/winget-pkgs/tree/master/manifests
-# Path: manifests/m/MoltenPixelStudio/LinkDrive/0.1.0/
-#
-# Bump the three version fields + InstallerUrl + InstallerSha256 on every
-# release. Can be automated via github-actions/winget-releaser.
-
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
 PackageIdentifier: MoltenPixelStudio.LinkDrive
 PackageVersion: 0.2.0
-PackageLocale: en-US
-Publisher: MoltenPixelStudio
-PublisherUrl: https://github.com/MoltenPixelStudio
-PublisherSupportUrl: https://github.com/MoltenPixelStudio/linkdrive/issues
-PackageName: LinkDrive
-PackageUrl: https://github.com/MoltenPixelStudio/linkdrive
-License: Proprietary
-LicenseUrl: https://github.com/MoltenPixelStudio/linkdrive/blob/main/LICENSE
-ShortDescription: Fast cross-device file explorer with VPS + LAN peer access
-Description: |
-  LinkDrive is a fast file explorer for Windows with first-class SFTP / VPS
-  browsing, multi-select transfers with cancel/resume, folder mirroring,
-  bookmarks, SSH config import, and Tailscale peer discovery.
-Tags:
-  - file-manager
-  - sftp
-  - ssh
-  - file-transfer
-ReleaseNotes: See https://github.com/MoltenPixelStudio/linkdrive/releases/tag/v0.2.0
-ReleaseNotesUrl: https://github.com/MoltenPixelStudio/linkdrive/releases/tag/v0.2.0
-ManifestType: singleton
+DefaultLocale: en-US
+ManifestType: version
 ManifestVersion: 1.6.0
-
-Installers:
-  - Architecture: x64
-    InstallerType: exe
-    InstallerUrl: https://github.com/MoltenPixelStudio/linkdrive/releases/download/v0.2.0/LinkDrive-Setup-0.2.0-win-x64.exe
-    # Replace with actual SHA256 of the release asset on each bump:
-    InstallerSha256: ACF21D97E6329F88B708FA9BD54123B637A73A819D0DA73150E7A423E4213D26
-    InstallerSwitches:
-      Silent: --silent-install
-      SilentWithProgress: --silent-install
-    InstallerSuccessCodes:
-      - 0
-    Scope: user
-    InstallModes:
-      - silent
-      - silentWithProgress
-      - interactive
-    UpgradeBehavior: install
-    ProductCode: com.moltenpixel.linkdrive

